go/stellar/global.go

```diff
@@ -27,7 +27,9 @@ func ServiceInit(g *libkb.GlobalContext, walletState *WalletState, badger *badge
 	if g.Env.GetRunMode() != libkb.ProductionRunMode {
 		stellarnet.SetClientAndNetwork(horizon.DefaultTestNetClient, build.TestNetwork)
 	}
-	g.SetStellar(NewStellar(g, walletState, badger))
+	s := NewStellar(g, walletState, badger)
+	g.SetStellar(s)
+	g.AddLogoutHook(s, "stellar")
```

Comment:
cc @mlsteele @patrickxb is there anything we want to clear in stellar on a dbnuke? from a quick pass I didn't see anything obvious

Reply:
stellar doesn't use leveldb.

Reply:
@patrickxb goal is to have the dbnuke hook clear any disk/memory caches not just leveldb

Reply:
then you would want to do what happens in service/main.go:OnLogout:

            log("resetting wallet state on logout")
            if d.walletState != nil {
                    d.walletState.Reset(m)                                                    
            }

on that Service.walletState variable.